cmd/juno/dbcmd.go

```diff
@@ -0,0 +1,202 @@
+	if err == nil {
+		info.L1Height = l1Head.BlockNumber
+		info.L1BlockHash = l1Head.BlockHash
+		info.L1StateRoot = l1Head.StateRoot
+	} else {
+		fmt.Printf("Failed to get the latest L1 block information: %v\n", err)
+	}
+
+	jsonData, err := json.Marshal(info)
+	if err != nil {
+		return fmt.Errorf("marshal JSON: %w", err)
+	}
+
+	fmt.Println(string(jsonData))
+
+	return nil
+}
+
+func dbSize(cmd *cobra.Command, args []string) error {
+	dbPath, err := cmd.Flags().GetString(dbPathF)
+	if err != nil {
+		return err
+	}
+
+	if dbPath == "" {
+		return fmt.Errorf("--%v cannot be empty", dbPathF)
+	}
+
+	if _, err = os.Stat(dbPath); os.IsNotExist(err) {
+		fmt.Println("Database path does not exist")
+		return nil
+	}
+
+	pebbleDB, err := pebble.New(dbPath)
+	if err != nil {
+		return err
+	}
+
+	var (
+		totalSize  utils.DataSize
+		totalCount uint
+
+		withHistorySize    utils.DataSize
+		withoutHistorySize utils.DataSize
+
+		withHistoryCount    uint
+		withoutHistoryCount uint
+
+		items [][]string
+	)
+
+	for _, b := range db.BucketValues() {
+		fmt.Fprintf(cmd.OutOrStdout(), "Calculating size of %s, remaining buckets: %d\n", b, len(db.BucketValues())-int(b)-1)
+		bucketItem, err := pebble.CalculatePrefixSize(cmd.Context(), pebbleDB.(*pebble.DB), []byte{byte(b)})
+		if err != nil {
+			return err
+		}
+		items = append(items, []string{b.String(), bucketItem.Size.String(), fmt.Sprintf("%d", bucketItem.Count)})
+
+		totalSize += bucketItem.Size
+		totalCount += bucketItem.Count
+
+		if utils.AnyOf(b, db.StateTrie, db.ContractStorage, db.Class, db.ContractNonce, db.ContractDeploymentHeight) {
+			withoutHistorySize += bucketItem.Size
+			withHistorySize += bucketItem.Size
+
+			withoutHistoryCount += bucketItem.Count
+			withHistoryCount += bucketItem.Count
+		}
+
+		if utils.AnyOf(b, db.ContractStorageHistory, db.ContractNonceHistory, db.ContractClassHashHistory) {
+			withHistorySize += bucketItem.Size
+			withHistoryCount += bucketItem.Count
+		}
+	}
+
+	table := tablewriter.NewWriter(os.Stdout)
+	table.SetHeader([]string{"Bucket", "Size", "Count"})
+	table.AppendBulk(items)
+	table.SetFooter([]string{"Total", totalSize.String(), fmt.Sprintf("%d", totalCount)})
+	table.Render()
+
+	tableState := tablewriter.NewWriter(os.Stdout)
+	tableState.SetHeader([]string{"State", "Size", "Count"})
+	tableState.Append([]string{"Without history", withoutHistorySize.String(), fmt.Sprintf("%d", withoutHistoryCount)})
+	tableState.Append([]string{"With history", withHistorySize.String(), fmt.Sprintf("%d", withHistoryCount)})
+	tableState.Render()
+
+	return nil
+}
+
+func getNetwork(head *core.Block, stateDiff *core.StateDiff) string {
+	networks := map[string]*utils.Network{
+		"mainnet":             &utils.Mainnet,
+		"sepolia":             &utils.Sepolia,
+		"goerli":              &utils.Goerli,
+		"goerli2":             &utils.Goerli2,
+		"integration":         &utils.Integration,
+		"sepolia-integration": &utils.SepoliaIntegration,
+	}
```

Comment:
you don't need map here, just `[]utils.Network{ ...}` name of a network can be retrieved through `Name` field